### PR TITLE
Laravel 5.6 support

### PR DESCRIPTION
**Pull request by fragkp**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,14 +18,9 @@ matrix:
...
       env: DEPS=dev
 
-    - php: 7.0
+    - php: 7.1.3
       env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_PHPUNIT_VERSION=5.7 SYMFONY_DEPRECATIONS_HELPER=23
 
-    - php: 7.1
-      env: LTS_VERSION=2 SYMFONY_DEPRECATIONS_HELPER=23
-    - php: 7.1
-      env: LTS_VERSION=3
```
Comment (wouterj) [inline .travis.yml]: Why did you remove these two? Is there something that blocks support for Symfony 2.8 and 3.4?

**Comment by wouterj** [comment]
Thanks for your PR btw! :smiley_cat: Great to see that we support Laravel 5.6 without changing anything in the PHP code.

Is there a reason you dropped Symfony 2 and 3 support? This bundle follows the Symfony practices more than the Laravel practices. So if we can keep support for older, still maintained, versions, we should try to do that. In this case, as this PR doesn't have PHP code fixes, we can also require either Laravel 5.5 or 5.6 in `composer.json`. (I'm not sure if 5.6 only supports Symfony 4?)

**Comment by fragkp** [comment]
Hey!

The reason behind dropping Symfony 2 & 3 is that Laravel 5.6 uses Symfony 4 packages.
I think, if you mix these dependencies it results in conflicts.

**Comment by wouterj** [comment]
Hi! I picked your commits and added a custom commit keeping Laravel 5.5 support, besides 5.6: https://github.com/wouterj/WouterJEloquentBundle/pull/85 Thanks for starting this PR!
